tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -113,18 +113,21 @@ describe( 'Shopper → Active Filters Block', () => {
 				}
 			);
 
+			await page.waitForSelector(
+				selectors.frontend.productsList + '.is-loading',
+				{ hidden: true }
+			);
+
 			await expect( page ).toClick( 'label', {
 				text: 'In stock',
 			} );
 
-			await page.waitForSelector( block.class );
+			await page.screenshot( {
+				path: 'aaa.png',
+				fullPage: true,
+			} );
 
```

Comment:
This has probably been forgotten, I assume? 🤔

Reply:
ahahah, yes! Powerful debug :P